frontend/src/scenes/pipeline/pipelinePluginConfigurationLogic.tsx

```diff
@@ -99,8 +105,7 @@ export const pipelinePluginConfigurationLogic = kea<pipelinePluginConfigurationL
                         defaultConfigForPlugin(values.plugin),
                         config
                     )
-                    // NOTE: getPluginConfigFormData already appends enabled
```

Comment:
this removed the double adding of enabled, but now it wasn't possible to use the toggle - it would always be false here, because enabled is pulled out from formdata in line 96